pages/common/php-artisan.md

```diff
@@ -0,0 +1,19 @@
+# php artisan
+
+> Laravel's Artisan command line interface.
+
+- Display a list of all available commands:
+
+`php artisan help`
+
+- Start PHP's built-in web server for the current Laravel application:
+
+`php artisan serve`
+
+- Start an interactive PHP command line interface:
```

Comment:
What's the different between this and `php -a`?

Reply:
Tinker is a REPL Laravel offers that gives an interactive interface with specific integrations (such as bootstrapping the framework), it is different to the `php -a` tool offered by PHP's native CLI.

Reply:
Ah, ok - thanks for clearing that up :-)